doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,35 @@
-=======================
-Release Build Creation
-=======================
+=========================================================
+Compile & Build Complete Cortx Stack Via Docker Container
+=========================================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
 
-   - You can use a Virtual Machine (VM) also.
+  - Please create VM(s) with at least 1 cores (vCPU) and 4GB of RAM.
+  - Ensure the VM is created with minimum of total 2 attached raw disks (additional disks apart from main bootable disk) per node
+
+    - Data Disks - Min 1 Disks per node (Capacity 20G+)
+    - Metadata Disks - Min 1 Disks per node (Capacity - 10% of total Data Disk Size)
+  - Ensure the VM is created with minimum 3 network Interfaces
+  - Ensure IP’s have assigned to all NICs e.g. eth33, eth34 and eth35
+  
+**Note:** Your NIC's may have different names so select the interfaces as per your setup
+ 
+  - Ensure system have valid hostname and they are accessible (i.e. ping)
```

Comment:
you can also write this as:

Ensure that the system have valid hostname and are accessible using ping.

Reply:
done